scripts/build.py

```diff
@@ -542,10 +557,9 @@ def save_hashes_as_json(target_filepath, file_hashes):
 
     file_hash = generate_md5_hash(target_filepath)
     relative_filepath = os.path.relpath(
-        target_filepath, os.path.join(os.path.curdir, 'build'))
+        target_filepath, os.path.join(os.path.curdir, 'build', 'assets'))
```

Comment:
Just to check, why did this change?

Reply:
The file path keys of `file_hashes` must be relative to _/build_ subfolders (_/build/assets, /build/extensions_) but the key for _hashes.js_ was relative to _/build_ so it was wrong. It didn't break anything because the whole path got replaced inside HTML, now it breaks because when replacing the file paths inside HTML files we add /assets (or /extensions etc.) before the to be replaced file path.

Reply:
Hm ok. Add comment within the file to explain that the filepath keys must be relative to /build subfolders and why.

Reply:
Done.